modules/product/src/Plugin/Field/FieldWidget/ProductVariationSkuWidget.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Drupal\commerce_product\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\Plugin\Field\FieldWidget\StringTextfieldWidget;
+
+/**
+ * Plugin implementation of the 'commerce_auto_sku' widget.
+ *
+ * @FieldWidget(
+ *   id = "commerce_auto_sku",
+ *   label = @Translation("Commerce auto SKU"),
+ *   field_types = {
+ *     "string"
+ *   }
+ * )
+ */
+class ProductVariationSkuWidget extends StringTextfieldWidget {
```

Comment:
Would it be best if we put this into its own issue to deprecate the contrib module? Allows us to write better test coverage, too